packages/p2p/src/components/buy-sell/buy-sell-form/buy-sell-form-receive-amount.tsx

```diff
@@ -7,21 +7,19 @@ import { useStores } from 'Stores';
 
 const BuySellFormReceiveAmount = () => {
     const { buy_sell_store } = useStores();
+    const { advert, is_sell_advert, receive_amount } = buy_sell_store;
 
     return (
-        <div className='buy-sell__modal-receive-amount'>
+        <div className='buy-sell-form__receive-amount'>
             <Text as='p' color='less-prominent' line_height='m' size='xxs'>
```

Comment:
we must remove thissss since its already the default 🙏 

```suggestion
            <Text as='p' color='less-prominent' size='xxs'>
```